syfertext/pipeline/single_label_classifier.py

```diff
@@ -408,6 +408,9 @@ def load_state(self) -> None:
         elif isinstance(result, State):
             state = result
 
+        elif isinstance(result, tuple):
```

Comment:
can you add comment why you are doing this

Reply:
Yeah so the thing is, search_resource is in the utils folder and almost all objects have utils as a dependency. If I take state as a dependency in utils (for the detail function), it creates a circular dependency, hence I am forced to do it this way